public/projects/zorkington.md

```diff
@@ -55,7 +55,7 @@ function moveToRoom(newRoom) {
 }
 ```
 
-You may want to create a [State Machine](/lessons/cs/state_machines) to represent the rooms.
+You may want to create a [State Machine](/lessons/javascript/state_machines) to represent the rooms.
```

Comment:
@Habenzy I think this link is broken.

Reply:
@Habenzy Yes, this lesson was in the CS track and it seems to no longer be reachable.

Reply:
@Habenzy I'll make an issue for this so we can fix it later. I'll accept for now. We will need this StateMachine lesson to live somewhere.